packages/runtime/container-runtime/src/gc/gcHelpers.ts

```diff
@@ -84,3 +98,66 @@ export function generateSortedGCState(gcState: IGarbageCollectionState): IGarbag
 	}
 	return sortedGCState;
 }
+
+/**
+ * Before GC data was written at the root of the summary tree, individual GC blobs were written at each data store;s
```

Comment:
I'd preface this saying it's for back-compat.  Also, typo `data store;s` -> `data store's`

Reply:
done.